src/js/components/Form/stories/UnsolicitedFeedback.js

```diff
@@ -0,0 +1,136 @@
+import React, { useContext, useState } from 'react';
+import styled from 'styled-components';
+import {
+  Box,
+  Button,
+  Form,
+  Layer,
+  FormField,
+  Text,
+  TextArea,
+  Header,
+  Heading,
+  ThumbsRating,
+  ResponsiveContext,
+} from 'grommet';
+
+// floating button purposes
+const PositionedFeedbackBox = styled(Box)`
+  bottom: 0px;
+  right: 0px;
+  position: ${(props) => props.theme.feedback?.buttonContainer?.position};
```

Comment:
Why does the story need to reach into the theme like this? I would have either expected that to be built in somehow or that those things shouldn't be in the theme at all.

Reply:
fixed